web/src/main/react-dashboard/package-lock.json

```diff
@@ -14,7 +14,7 @@
       "resolved": "https://registry.npmjs.org/accepts/-/accepts-1.3.4.tgz",
```

Comment:
There are major changes to this file. Can you verify and revert any unnecessary changes? I don't see any dashboard related changes in this pr.